src/types.js

```diff
@@ -73,6 +73,8 @@ export type ViewStyle = {
   shadowOffset: { width: number, height: number },
   shadowOpacity: number,
   shadowRadius: number,
+  shadowSpread: number,
+  shadowInner: "innerShadow" | "shadow",
```

Comment:
this should be a `boolean`.

We could probably reuse the `SketchShadow` type as well (`{...} & SketchShadow`)